apps/dashboard/redirects.js

```diff
@@ -270,11 +270,6 @@ function redirects() {
       destination: "/in-app-wallets",
       permanent: false,
     },
-    {
```

Comment:
why is this being removed?

Reply:
no redirect required, moved the <slug>/permissions/page.tsx to <slug>/page.tsx